hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/CustomizedScanInfoBuilder.java

```diff
@@ -42,13 +43,18 @@
 
   public CustomizedScanInfoBuilder(ScanInfo scanInfo) {
     this.scanInfo = scanInfo;
-    this.scan = new Scan();
+    try {
+      this.scan = new ImmutableScan(new Scan());
```

Comment:
This is used by tests only ?
Do u want to create ImmutableScan over a new Scan()?  Or direct new ImmutableScan is enough?

Reply:
I just provided 2 constructors to Scan i.e.
1) Scan(Scan scan)
2) Scan(Get get)

I believe, given that this is Immutable subclass, providing default constructor might not be much useful. Even if we give, it will internally use `this(new Scan())`. Hence, I thought of not providing default constructor at all. Thought?